mesheryctl/cmd/start.go

```diff
@@ -37,7 +37,9 @@ var startCmd = &cobra.Command{
 		// var stderr bytes.Buffer
 
 		if _, err := os.Stat(mesheryFolder); os.IsNotExist(err) {
-			_ = os.Mkdir(mesheryFolder, 0777)
+			if err := os.Mkdir(mesheryFolder, 0777); err != nil {
+				log.Fatal("[ERROR] Could not create Meshery directory. The error message: \n", err)
```

Comment:
Hey @supra08 the changes looks great, 
But there is some slight changes that are required  we are not required to print  "[ERROR]" in statements. log.fatal will itself do the work!
you just need to remove "[ERROR]" form statements.